src/components/Contact/ContactSponsorUs/ContactSponsorUs.css

```diff
@@ -10,18 +10,36 @@
   border: 1px solid #4F72B8;
   border-radius: 8px;
   font-size: 30px;
-  width: 22%;
+  width: 70%;
+  min-width: 220px;
   height: 80px;
   color: white;
   transition: 0.3s;
   background-color: #13191E;
 }
 
-@media (max-width: 750px) {
+.contact-translate-sponsor-us-btn {
+  border: 1px solid #4F72B8;
+  border-radius: 8px;
+  font-size: 20px;
```

Comment:
NIT: According to Googles' HTML/CSS Code Style guide, CSS attributes should be listed alphabetically.